src/opensynth/models/faraday/gaussian_mixture/fit_kmeans.py

```diff
@@ -59,7 +59,7 @@ def fit_kmeans(
     # )
     # trainer.fit(kmeans_module, data)
```

Comment:
Add a comment to say that we are replacing the PyTorch Lightning KMeans implementation with the sk-learn package. 
Add the following comment `#TODO : perform k-means on a random subsample of the training dataset to speed up convergence.`  i.e., in the future will perform k-means fitting on a randomised sample of the dataset, otherwise the k-means initialisation could take ages when we train on the very large NRD dataset. GMM will be trained on the full sample but distributed across GPUs to speed up the process.